services/records_to_knack.py

```diff
@@ -56,6 +54,18 @@ def is_equal(rec_src, rec_dest, keys):
     return all(tests)
 
 
+def remove_raw_tags(records):
+    """
+    Removes "_raw" from field names, if they exist.
+    """
+    fields_to_rename = [f for f in list(records[0].keys()) if "_raw" in f]
+    if fields_to_rename:
+        for rec in records:
+            for f in fields_to_rename:
+                rec[f.replace("_raw", "")] = rec.pop(f)
+    return records
+
+
```

Comment:
the email and name datatypes in Knack were giving me trouble so I allowed for these `_raw` fields to get passed along to the destination Knack app. This just removes the `_raw` from the field name so it'll get recognized by the API.

Reply:
so, some of the fields that you are moving between apps use the `_raw` tag, because they are special compound field types like names and emails. but when you update records in the destination app, you need to drop the `_raw` from the filed names in the record payload.

do i have that right? that makes sense. and this seems backwards compatible and fine. maybe add some more detail to the function description?

Reply:
yup, that's right. I'll add some more to the description.

Reply:
And this is indeed backwards compatible as no destination fields in field_maps.py have `_raw` in them besides this new ETL.